RuGenerous/rugenerous-frontend
# Feature -> price data fetch without refreshing page

## Issue Link
[Trello Board Issue](https://trello.com/c/TkgJJ5KJ)

## Description
This pull request adds the react-query library to make it easy to handle server state.

As a draft PR, this currently only updates the way price is fetched and displayed on the dashboard view.

The price is refetched every 10 seconds. This interval can be adjusted in the constants file for intervals.

## How To Review This
 - Pull branch
 - run ```yarn``` 
 - run ```yarn start```

Observe the price on the dashboard.
Observe the react-query devtools ( the weird icon on the lower left, click to open )

the react query devtools only show on non production environment

## Checklist:

## References
[React Query Documentation](https://react-query.tanstack.com/examples/auto-refetching)
